rosidl_generator_tests/test/rosidl_generator_c/test_interfaces.c

```diff
@@ -45,21 +45,16 @@
   u"Deep into that darkness peering, long I stood there wondering, fearing \u2122"
 #define ARR_SIZE 3
 
-#define STRINGIFY(x) _STRINGIFY(x)
 #define _STRINGIFY(x) #x
+#define STRINGIFY(x) _STRINGIFY(x)
 
-#define EXPECT_FALSE(arg) if (arg) { \
-    fputs(STRINGIFY(arg) " is not false\n", stderr); \
-    return 1; \
-}
-#define EXPECT_TRUE(arg) if (!(arg)) { \
-    fputs(STRINGIFY(arg) " is not true\n", stderr); \
-    return 1; \
-}
-#define EXPECT_EQ(arg1, arg2) if ((arg1) != (arg2)) { \
-    fputs(STRINGIFY(arg1) " != " STRINGIFY(arg2) "\n", stderr); \
-    return 1; \
-}
+#define PUTS(arg, extra) fprintf(stderr, "%s%s\n", STRINGIFY(arg), extra)
```

Comment:
Is that better to have space in `PUTS`, instead of `EXPECT_xxx` condition check?

```suggestion
#define PUTS(arg, extra) fprintf(stderr, "%s %s\n", STRINGIFY(arg), extra)
```

Reply:
I think it is fine either way.  Given that I've already run CI on this, and it is just in a test, I'm going to go with what is in here for now.  We can follow-up later if we want to make that change.